canonical/jp.ubuntu.com
# Add /certified page

## Done

- Added new /certified page
- Figma: https://www.figma.com/design/tI8YK4wQnKiWLAxdX1fWnB/24.10-jp.ubuntu.com?node-id=0-1&t=Ey9MQHs6ERH0AKQV-0
- Docs: https://docs.google.com/document/d/1aNshSU4yZ8otlPoWTjHmO8lq1gY92J1T4w5iVK3pzdI/edit

## QA

- Go to [demos](https://jp-ubuntu-com-482.demos.haus/certified)
- Check that the /certified webpage adheres to design and docs

## Issue / Card

Fixed [WD-10813](https://warthogs.atlassian.net/browse/WD-10813)

## Screenshots

[if relevant, include a screenshot]


[WD-10813]: https://warthogs.atlassian.net/browse/WD-10813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
